targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala

```diff
@@ -100,33 +99,28 @@ object TargetBody extends ModelOptics {
           val searchAndSet: NonEmptyString => Callback =
             searchAndGo(modify.andThen(_.runInCB))
 
-          Grid(columns = Three,
-               clazz = ExploreStyles.FullHeightWidth,
-               stretched = true,
-               padded = GridPadded.Horizontally
-          )(
-            GridRow(stretched = true)(
-              GridColumn(stretched = true, computer = Four)(
+          <.div(
```

Comment:
Can this be converted to `ReactFragment`?

Reply:
Sounds like it. I'll look into it.